internal/cluster/kube/servicesinformer_test.go

```diff
@@ -0,0 +1,49 @@
+package kube
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"testing"
+)
+
+func Test_parseService(t *testing.T) {
+	testCases := []struct {
+		svc       interface{}
+		errWanted bool
```

Comment:
comment - better to check exact errors versus error true/false. errWanted error

Reply:
I'd actually like to discuss about this one, I have been wondering that too and I saw that some `assert` function check it too. But does it mean the message need to be the same too?